README.md

```diff
@@ -40,14 +41,15 @@ EPyQ widgets should be visible at the bottom of the widget box on the left.
 - Install Python 3.7
   - Consider [pyenv](https://github.com/pyenv/pyenv) to get Python versions
 - Install git
-- `git clone https://github.com/altendky/st`
+- `git clone https://github.com/epcpower/st`
 - `cd st`
 - `git submodule update --init`
-- `python boots.py ensure`
+- `poetry install`
+- `poetry run buildui`
```

Comment:
i am adding another manual step that was initially called in the setup.py.  not sure how to get this to run automatically.